devops/docker-compose.local.yml

```
@@ -9,16 +9,21 @@ networks:
 services:
 
   rabbitmq:
-    image: rabbitmq:3.12.6
+    image: rabbitmq:3-management
     networks:
       - geoapi
     environment:
       - RABBITMQ_DEFAULT_USER=dev
       - RABBITMQ_DEFAULT_PASS=dev
       - RABBITMQ_DEFAULT_VHOST=dev
+    healthcheck:
+      test: rabbitmq-diagnostics -q ping
+      interval: 5s
+      timeout: 30s
+      retries: 5
 
   postgres:
-    image: mdillon/postgis:11-alpine
+    image: postgis/postgis:11-3.0-alpine
```

```suggestion
    image: postgis/postgis:11-3.3-alpine
```

nice.  i see we have moved away from mdillon/postgis on the [production server.](https://github.com/TACC-Cloud/geoapi/blob/main/devops/database/docker-compose-database.yml#L3)

---

Yes I meant to confirm - good change to move to official postgis repo in your opinion?

---

and deployed we got `postgis/postgis:11-3.3-alpine` insetad of the 3.0.  so we can match that here too.